lib/OpenQA/WebAPI/Plugin/Gru.pm

```diff
@@ -61,7 +61,7 @@ sub register {
     $conn->dsn($self->dsn());
 
     # set the search path in accordance with the test setup done in OpenQA::Test::Database
-    if (my $search_path = $ENV{TEST_PG_SEARCH_PATH}) {
+    if (my $search_path = $self->schema->tmp_schema) {
```

Comment:
I like that you removed my hack to use an environment variable here. However, I liked my variable name more. Or maybe call it `search_path_for_tests`?

Reply:
I hate both equally. 🤣

Reply:
I picked `tmp_schema` because `OpenQA::Test::Database` used `$schema->{tmp_schema}`.